modules/images-configuration-registry-mirror-configuring.adoc

```diff
@@ -121,7 +121,7 @@ spec:
 <9> Optional: Indicates a registry, which allows you to use any image in that registry. If you specify a registry name, the object is applied to all repositories from a source registry to a mirror registry.
 <10> Pulls the image `registry.example.com/example/myimage@sha256:...` from the mirror `mirror.example.net/image@sha256:..`.
 <11> Pulls the image `registry.example.com/example/image@sha256:...` in the source registry namespace from the mirror `mirror.example.net/image@sha256:...`.
-<12> Pulls the image `registry.example.com/myimage@sha256` from the mirror registry `example.net/registry-example-com/myimage@sha256:...`.
+<12> Pulls the image `registry.example.com/myimage@sha256` from the mirror registry `mirror.example.net/registry-example-com/myimage@sha256:...`.
```

Comment:
I see the `...`  ending does exist with surrounding entries but is it needed?

How about:

mirror.example.net/registry-example-com/myimage@sha256:<sha_number>

Reply:
myimage > <image> ?

Reply:
I'm not sure if  the `...` is needed. Should I go back and ask

Reply:
Otherwise we can go with your update

Reply:
No harm asking for clarification, I think the ellipsis might mean <sha_number> but it's not called out in the same way. Maybe the ellipsis covers a few additional values.